workers/convert/pdf/tasks.py

```diff
@@ -8,6 +8,7 @@
 
 
 class SplitPdfTask(BaseTask):
+
```

Comment:
Ich bin gegen so viele Leerzeilen. Ich zitiere PEP8: "Extra blank lines may be used (sparingly)" und "Use blank lines in functions, sparingly..."

Reply:
Mir eigentlich egal, wir sollten es nur einheitlich machen.